plugins/nginx-vhosts/install

```diff
@@ -49,18 +49,20 @@ gpasswd -M "" dokku
 
 case "$DOKKU_DISTRO" in
   debian)
-    /usr/sbin/invoke-rc.d nginx start
+    NGINX_INIT="/usr/sbin/invoke-rc.d nginx"
     ;;
 
   ubuntu)
-    /etc/init.d/nginx start
+    NGINX_INIT="/etc/init.d/nginx"
     ;;
 
   opensuse)
-    /sbin/service nginx start
+    NGINX_INIT="/sbin/service nginx"
     ;;
 
   arch)
-    /usr/bin/systemctl start nginx
+    NGINX_INIT="/usr/bin/systemctl nginx"
```

Comment:
This won't work because `systemctl` requires that the command be before the program.